rskj-core/src/main/java/org/ethereum/config/blockchain/DevNetConfig.java

```diff
@@ -69,4 +69,9 @@ public boolean isRfs94() {
     public boolean isRcs230() {
         return true;
     }
+
+    @Override
+    public boolean isRskIp15Bis() {
+        return true;
```

Comment:
Shouldn't this be `false`? Deploying this code would make this immediately apply the network upgrade code.

Same for other network configurations.